packages/portfolio/src/portfolio-routes.tsx

```diff
@@ -1,17 +1,23 @@
-import { PortfolioRoutesLoaded } from './portfolio-routes-loaded.tsx'
-import { PortfolioUiAccountGuard } from './ui/portfolio-ui-account-guard.tsx'
-import { PortfolioUiNetworkGuard } from './ui/portfolio-ui-network-guard.tsx'
+import type { Account } from '@workspace/db/entity/account'
+import type { Network } from '@workspace/db/entity/network'
+import { useDbAccountActive } from '@workspace/db-react/use-db-account-active'
+import { useDbNetworkActive } from '@workspace/db-react/use-db-network-active'
+import { useRoutes } from 'react-router'
+import { PortfolioFeatureIndex } from './portfolio-feature-index.tsx'
+import { PortfolioFeatureTx } from './portfolio-feature-tx.tsx'
+
+export interface AccountNetwork {
+  account: Account
+  network: Network
+}
 
 export default function PortfolioRoutes() {
-  return (
-    <PortfolioUiNetworkGuard
-      render={({ network }) => (
-        <PortfolioUiAccountGuard
-          render={({ account }) => {
-            return <PortfolioRoutesLoaded account={account} network={network} />
-          }}
-        />
-      )}
-    />
-  )
+  const account = useDbAccountActive()
+  const network = useDbNetworkActive()
```

Comment:
This is really clean, love it. 

I'm wondering if we even need to call these hooks here and pass it into `PortfolioFeatureIndex` and `PortfolioFeatureTx` or instead call the hooks directly in those features.

Reply:
We don't. I wanted to handle removing the prop drilling in another PR.